qutebrowser/config/configdata.yml

```diff
@@ -978,6 +978,13 @@ keyhint.blacklist:
     Globs are supported, so `;*` will blacklist all keychains starting with `;`.
     Use `*` to disable keyhints.
 
+keyhint.radius:
+  type:
+    name: Int
+    minval: 0
+  default: 6
+  desc: The rounding radius for the edges of the keyhint dialog.
```

Comment:
suggestion: add the unit (px) to the description

Reply:
I think this is a good idea. But at the same time there are a lot of configuration settings that don't have unit descriptions. Including c.prompt.radius, from which I based my configuration. Perhaps making a pull request with all the configurations fixed in a consistent manner would be better?